reports/summary-2020.md

```diff
@@ -696,7 +696,7 @@ initiative with supporting links to KEPs and more.
     - Kubefed - support for pull-based reconciliation 
     - Work API - road to alpha
 
-- [SIG Network](https://git.k8s.io/community/sig-network/annual-report-2020.md#current-initiatives-and-project-health)
+- [SIG Network](https://git.k8s.io/community/sig-network/annual-report-2021.md#current-initiatives-and-project-health)
```

Comment:
```suggestion
- [SIG Network](https://git.k8s.io/community/sig-network/annual-report-2020.md#current-initiatives-and-project-health)
```

This should be fixed in k/community

Reply:
Sounds good, I'll create an issue there 👍